### PR TITLE
Exclude unique post fields from duplication

### PR DESCRIPTION
**Pull request by pcfreak30**
#3563

### CODE REVIEW

File: classes/PodsAPI.php [hunk 1]
```diff
@@ -4201,6 +4201,18 @@ public function duplicate_pod_item ( $params ) {
         );
 
         foreach ( $fields as $field ) {
+	        if ( in_array( $field['name'], array(
+		        'ID',
+		        'post_name',
+		        'post_date',
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: I think we can remove post_date, post_date_gmt, post_modified, post_modified_gmt from this list.

Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Also I think we should probably do some `$pod->pod_data['type']` checks here, as well as generally skipping the field `if ( $pod->pod_data['field_id'] == $field['name'] )`.

**Comment by pcfreak30** [comment]
You need them as I tested and WP doesn't override it if it exists. So the post dates have to be skipped or they are copied direct. As for the other checks, I don't know much about them so you would need to fork my PR to patch it.

**Comment by sc0ttkclark** [comment]
@pcfreak30 can you give me write access to that repo?

**Comment by pcfreak30** [comment]
@sc0ttkclark you have been invited as a collaborator.

**Comment by jimtrue** [comment]
@sc0ttkclark This is still flagged for Review, is this one ready to be merged?

**Comment by sc0ttkclark** [comment]
Cleaned up the logic so we can expand to cover any other cases in which certain fields need to be ignored and added a filter to future proof that further.

*Future: Add coverage for duplicating users, terms, and comments.*

Also cleaned up formatting while I was there since the method was small.

**Comment by sc0ttkclark** [comment]
Just added some more coverage for terms, users, and comments.

**Comment by sc0ttkclark** [comment]
Once this passes tests, we can merge.

**Comment by pcfreak30** [comment]
Just realized you excluded ID from the ignore post_type array. it must be included as it will run an update on wp_insert_post/wp_update_post and not a new item. Found this bugging my code in the dev site.

**Pull request merged**